src/ggrc/assets/javascripts/plugins/canjs_extensions.js

```
@@ -79,6 +79,10 @@
 
   // Turn camelSpace strings into Camel Space strings
   can.spaceCamelCase = function (string) {
+    if (!_.isString(string)) {
+      throw new TypeError('Invalid type, string required.');
+    }
```

I like this. Raising an error instead of silently converting non-strings is IMO better, because it might reveal bugs in the code. It might happen, for example, that some, presumably, string value is not properly initialized, and we would know immediately.

Although it could be argued that for `null` or `undefined` values, an empty string could be returned. What are the current typical use cases for this function?